### ISSUE TITLE
Chromium icon is shown on Autofill modal

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#40779]
### Description

Chromium icon is shown on Autofill modal

### Steps to reproduce

1. Save credit card info
2. Go to any page where card autofill can be triggered
3. Autofill modal shows Chromium icon

### Actual result

<img width="540" alt="image" src="https://github.com/user-attachments/assets/46ccfb49-daab-4641-9059-cc1b5f820200">

### Expected result

No Chromium branding 

### Reproduces how often

Easily reproduced

### Brave version

1.69.157

### Device

- Brand/model: Google Pixel 7
- Android version: 14

### Channel information

- [X] release (stable)
- [X] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc: @aguscruiz should the button be in red or changed to Blurple

@brave/qa-team @deeppandya

**Comment by aguscruiz** [comment]
Yeah, should be our filled-blurple and not the fully rounded button we see here 

Simple mock from our template component in the design library
https://www.figma.com/design/loFHVWcBAuuApFsH9V3T5I/%F0%9F%A4%96-Android?node-id=2863-1856&t=xyn33upAvFU9Gxkq-1

![image](https://github.com/user-attachments/assets/ccba1199-dc14-4c75-ae24-41dcd3ffe5c1)

**Comment by srirambv** [comment]
Couple of other screens that requires changing of buttons. 

<img width="540" alt="image" src="https://github.com/user-attachments/assets/3c2bac42-91af-4788-bfbb-8727298d1fb7"> | <img width="540" alt="image" src="https://github.com/user-attachments/assets/92ca3ef3-a9ed-4597-a5ed-5389f53a0ed3">
-- | --

We'll also need to change graphics on the first screenshot which seems to be Chromium icon
